Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -375,15 +389,21 @@ function getRetryIntervalInSeconds(retryCount: number): number {
     return exponentialBackOff < MaxRetryLimitInSeconds ? exponentialBackOff : MaxRetryLimitInSeconds ;
 }
 
-async function downloadZip(artifactArchiveUrl: string, downloadPath: string, zipLocation: string, handler: webHandlers.PersonalAccessTokenCredentialHandler, downloaderOptions: engine.ArtifactEngineOptions) {
+async function downloadZip(artifactArchiveUrl: string, downloadPath: string, zipLocation: string, handler: webHandlers.PersonalAccessTokenCredentialHandler, downloaderOptions: engine.ArtifactEngineOptions, checkDownloadedFiles: boolean = false) {
```

Comment:
Could you please break this line a little bit?

Reply:
Fixed it.